source/patterns/@aws-solutions-constructs/aws-eventbridge-lambda/README.md

```diff
@@ -82,7 +85,7 @@ Out of the box implementation of the Construct without any override will set the
 * Enable reusing connections with Keep-Alive for NodeJs Lambda function
 * Enable X-Ray Tracing
 * Set Environment Variables
-  * AWS_NODEJS_CONNECTION_REUSE_ENABLED (for Node 10.x and higher functions)
+* AWS_NODEJS_CONNECTION_REUSE_ENABLED (for Node 10.x and higher functions)
```

Comment:
Not an issue here, but when we write aws-lambda-eventbridge let's remember to set an environment variable on the Lambda function with the identity of the event bus.

Reply:
ok, point taken